source/index.js

```diff
@@ -98,15 +98,22 @@ class Popover extends React.Component {
   }
   componentDidUpdate(propsPrev, statePrev) {
     //log(`Component did update!`)
+    const willOpen = !propsPrev.isOpen && this.props.isOpen
```

Comment:
this is the only actual change. See 457401b3333d34795cbe292746727e06a5a72642 for the original change that doesn't go far enough.